website/integrations/services/nextcloud/index.md

```diff
@@ -29,10 +29,10 @@ The following placeholders will be used:
 
 Create an application in authentik and note the slug you choose, as this will be used later. In the Admin Interface, go to Applications->Providers. Create a SAML provider with the following parameters:
 
--   ACS URL: `https://nextcloud.company/apps/user_saml/saml/acs`
+-   ACS URL: `https://nextcloud.company/index.php/apps/user_saml/saml/acs`
```

Comment:
I think this should be a note underneath instead of changing the URLs, as this seems to depend on the nextcloud installation and if URL rewrites are enabled, something like this below the parameters:

```markdown
:::note
Depending on your nextcloud configuration, you might need to use `https://nextcloud.company/index.php/` instead of `https://nextcloud.company/`
:::
```